### ISSUE TITLE
Account omega/alpha based on PLEX donation notification?

### ISSUE THREAD

**Issue by douglas24** [eveseat/seat#437]
Is it possible to add a feature to use the notifications characters get about PLEX donations to calculate whether the account is alpha/omega and also when the omega will expire?

**Comment by warlof** [comment]
Hi,

We don't have information about omega/alpha character.

Plex may be use for a lot of things which is not related to game time. Furthermore, it is not the only way to pay a subscription. 

I'm not sure about the feasibility of this.

**Comment by douglas24** [comment]
I know it's not the only way but it's one of the two main ways. Since the miscellaneous notification mails are all the same format with just the name of the character that donated the PLEX different, it shouldn't be too hard to parse? Like an if "has donated 30 days of Clone State Omega time to you." then "Sent: 2018.08.22 00:53" + 30 days = subscription end. Maybe not perfect but better than nothing since the API endpoint is gone.

**Comment by warlof** [comment]
So what about classic subscription case?
No notification?

**Comment by herpaderpaldent** [comment]
Imagine every meteor package or buy 1 get 2 month offers. They all do no notification. 

Only hack could be a running skill queue but that regards a character not an account

**Comment by herpaderpaldent** [comment]
i suggest to close this, as it can never reliable be fullfilled.
